src/System.Security.Principal.Windows/src/Microsoft/Win32/SafeHandles/SafeTokenHandle.cs

```diff
@@ -19,15 +19,13 @@
 namespace Microsoft.Win32.SafeHandles
 {
     [SecurityCritical]
-    internal sealed class SafeTokenHandle : SafeHandle
+    public sealed class SafeTokenHandle : SafeHandle
     {
         private const int DefaultInvalidHandleValue = 0;
```

Comment:
You left this definition, but `InvalidHandle` uses `IntPtr.Zero`.  You should probably pick one or the other (I feel like `IntPtr.Zero` reads a little cleaner).